en_US/changes/changes-ee-4.3.md

```diff
@@ -19,8 +19,16 @@ ref: undefined
 
 *Release Date: 2022-02-11*
 
+### Important
+
+We fixed a bug in the calculation of the total number of license connections in 4.3.7, the license will correctly check the total number of connections in the cluster instead of only the number of connections on each node by mistake.
```

Comment:
```suggestion
We fixed a bug in the calculation of the total number of license connections in 4.3.7, the license will correctly check the total number of connections from the entire cluster instead of only the number of connections each node locally.
```